src/Illuminate/Validation/Validator.php

```diff
@@ -1828,6 +1828,32 @@ protected function validateRegex($attribute, $value, $parameters)
     }
 
     /**
+     * Validate that an attribute contains at least one of a list of list of substrings.
+     *
+     * @param $attribute
+     * @param $value
+     * @param $parameters
+     * @return bool
+     */
+    protected function validateContains($attribute, $value, $parameters)
+    {
+        return str_contains($value, $parameters);
+    }
+
+    /**
+     * Validate that an attribute contains none of a list of list of substrings.
+     *
+     * @param $attribute
+     * @param $value
+     * @param $parameters
+     * @return bool
+     */
+    protected function validateNotContains($attribute, $value, $parameters)
+    {
+        return ! str_contains($value, $parameters);
```

Comment:
Can't we also do something like this here:

    return ! $this->validateContains($attribute, $value, $parameters);